src/map/skill.c

```diff
@@ -13214,95 +13468,98 @@ static struct skill_unit_group *skill_unitsetting(struct block_list *src, uint16
 		case DC_UGLYDANCE:
 			val1 = 10; //FIXME: This value is not used anywhere, what is it for? [Skotlex]
 			break;
+
+#ifndef RENEWAL
```

Comment:
I am not sure whether I should add some toggle to rollback to previous music/dance behavior for RE. The new effects requires 2018-11+ clients or characters will show no visual effect.

But making a toggle for that much of changes is... complicated. I don't know where to set the boundaries, and we have several conf files to change, not only skill_db.

Reply:
While I was originally in favor of keeping the option to revert these rebalance changes to support older clients, I realize it's too much of a burden (in both implementation and future maintenance), so I'll take it back.

Considering that I expect a lot of people running renewal servers to have adopted 2018 or newer clients by now (if anything, for the UI improvements they bring), and that it's completely out of the question to have a third "pre-rebalance" build mode and db folder to maintain, I'm in favor of requiring a 2018-11+ client for renewal.

People that wish to run a renewal server on an older client (or want to avoid the rebalance changes), can revert these changes on their fork (or patch their clients if someone comes up with client backport patches). And people that want to include some of these rebalance changes in their pre-renewal servers, may also do so in their fork.

Let's keep it rolling - no reverts for renewal, while pre-renewal stays as it is without the rebalance changes